lib/media/stall_detector.js

```diff
@@ -141,6 +142,9 @@ shaka.media.StallDetector.MediaElementImplementation = class {
 
   /** @override */
   shouldBeMakingProgress() {
+    // If we are in WebOS platform disable StallDetector
```

Comment:
Please add a brief explanation of why we do this.  Feel free to correct or rephrase this as you like, but my understanding from #1704 is this:

```js
    // If we are in WebOS platform disable StallDetector.  This is because WebOS
    // TVs tend to have long hardware pipelines that respond slowly, so we tend
    // to mistake slow operation for a stall.
```